templates/config.py

```diff
@@ -7,6 +7,8 @@
 TIAAS_SEND_EMAIL_TO = '{{ tiaas_info.owner_email }}'
 TIAAS_SEND_EMAIL_FROM = "tiaas+noreply@example.org"
 
+TIAAS_SHOW_ADVERTISING = {{ tiaas_show_advertising }}
```

Comment:
```suggestion
TIAAS_SHOW_ADVERTISING = {{ tiaas_show_advertising | ternary('True', 'False') }}
```

and ok, probably a lot more that should be done, but that's something for me.